Yoast/javascript
# Add passive voice assessment for Portuguese

## Summary
This PR can be summarized in the following changelog entry:
* [yoastseo] Adds a passive voice assessment for Portuguese.
* [Yoast SEO Free] Improves the Portuguese readability analysis by adding the passive voice assessment.

## Relevant technical choices:
* The word list for the Portuguese participles contains all words we want to match and consists of the most commonly used participle forms. 

## Test instructions
This PR can be tested by following these steps:
* Run yarn test and make sure all current specs pass
* In the content analysis app:
1. Run the content analysis app and set the locale to `pt_PT`.
2. Write four Portuguese sentences that have a passive voice, e.g.  _`A decisão será aprovado. Cada participante foi informado. A nova regra será integrada. Todas as regras foram protocoladas durante a reunião.`_
The passive voice assessment must conclude that too much passive voice is used.
3. Write four Portuguese sentences that have active voice, e.g. 
  _`Eu aprovo esta decisão. O líder informou todos os participantes da reunião. Eles tomaram a decisão quando todos estavam presentes.  É o método preferido de vários membros da equipe de produção.`_ 
The passive voice assessment must conclude that enough active voice is used.

Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-479
